README.md

```diff
@@ -104,7 +104,33 @@ curl https://raw.githubusercontent.com/estafette/estafette-gke-preemptible-kille
 curl https://raw.githubusercontent.com/estafette/estafette-gke-preemptible-killer/master/kubernetes.yaml | envsubst | kubectl apply -n ${NAMESPACE} -f -
 ```
 
+### Deploy with Kustomize
+
+Create a `kustomization.yaml` file:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+commonLabels:
+  app: preemptible-killer
+bases:
+- github.com/coryodaniel/estafette-gke-preemptible-killer//manifests
```

Comment:
why is there dobule slash in here?

Reply:
it is the go-getter syntax for a [subdirectory](https://github.com/hashicorp/go-getter#subdirectories)